README.md

```diff
@@ -402,7 +424,7 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
     const name = 'Capt. Janeway';
     ```
 
-  - [6.2](#6.2) <a name='6.2'></a> Strings longer than 80 characters should be written across multiple lines using string concatenation.
+  - [6.2](#6.2) <a name='6.2'></a> Strings that cause the line to go over 100 characters should be written across multiple lines using string concatenation.
```

Comment:
do we want to encourage string templates instead of concatenation? also isn't our linter set up for 160 chars?